### ISSUE TITLE
Doesn't work with Arduino Nano 33 BLE 

### ISSUE THREAD

**Issue by Chanuka-Jayanath** [107-systems/107-Arduino-MCP2515#103]
<a href="https://107-systems.org/"><img align="right" src="https://raw.githubusercontent.com/107-systems/.github/main/logo/107-systems.png" width="15%"></a>

### :bug: Bug Report

**Describe the bug**

I am working on a project on which I use a MCP2515 Module with a 8Mhz crystal oscillator. Since I'm using an Arduino Nano 33 IoT board (because it supports BLE), I downloaded this library because it was compatible with the architecture of the board. After downloading and testing the given example, it worked well. But since my usage is only based on BLE and not Wi-Fi, I had an Arduino Nano 33 BLE board which has BLE 5.0 instead of BLE 4.2 on the IoT board. I checked if this library was compatible with the Arduino Nano 33 BLE on the github repo on which it indicates that it is compatible. So I compiled the example code and ran it. This is where I have an issue. The program compiles without any problem and get transfered to the board. But, when I look for the CAN messages with a CAN data recorder. I see a single iteration of CAN message. Since in the example, the message is looped every 100ms, I should be expecting new iterations of the CAN message every 100ms too. This expected behaviour is working fine the IoT board but not with the BLE board sadly.... I suppose it has to do with the library compatibility because, the pinout I use is the same for the boards and every components are well soldered with wires without any jumper wires. 

**Minimum example code to replicate the bug**
```C++
/* Please fill out*/
```

**Minimum configuration to replicate the bug**
Please provide the configuration data used to build your application:
* Build Toolchain: (e.g. Arduino IDE 1.8.13, arduino-cli 0.13.0)
* Arduino Core: (e.g. ArduinoCore-samd 1.8.9)
* Library Version (of all involved libraries): (e.g. 107-Arduino-TMF8801 1.1.0, 107-Arduino-MCP2515 1.2.0)

**Comment by aentinger** [comment]
Yes, sorry the README is misleading, since `mbed` support (and therefore support for the Arduino Nano 33 BLE has been removed in https://github.com/107-systems/107-Arduino-MCP2515/commit/632aa90a03861bc3058b5f54809626efe877161c - partially because it really never worked well). I'll also remove the notice of support in this repositories README.
